draft-ietf-tls-tls13.md

```diff
@@ -273,10 +273,6 @@ that has two basic properties:
 - The connection is reliable.  Messages include an authentication
   tag which protects them against modification.
 
-Note: The TLS Record Protocol can operate in an insecure mode but is generally
-only used in this mode while another protocol is using the TLS Record
-Protocol as a transport for negotiating security parameters.
-
```

Comment:
Oh, so that's worth putting in the *introduction*?  This is talking about integrity-only modes, which are dead.  Don't split hairs on me.

And I would contest the point that the handshake is insecure just because it is sent in the clear.  Conveying the necessary level of nuance to be able to understand this point isn't worth spending introductory text on.